src/users/users.controller.ts

```diff
@@ -75,6 +75,25 @@ export class UsersController {
     );
   }
 
+  @ApiOperation({ summary: 'Gets a specific User' })
+  @ApiParam({ description: 'Unique User graffiti', name: 'graffiti' })
+  @Get('/graffiti/:graffiti')
+  async findByGraffiti(
+    @Param(
+      'graffiti',
+      new ValidationPipe({
+        errorHttpStatusCode: HttpStatus.UNPROCESSABLE_ENTITY,
+      }),
+    )
+    graffiti: string,
+  ): Promise<SerializedUser> {
```

Comment:
nit:
```suggestion
  ): Promise<SerializedUserWithRank> {
```

I just realized the `find` method above should probably return this type too.